drivers/gpio/gpio_renesas_ra8.c

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT renesas_ra8_gpio
+
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/drivers/gpio/gpio_utils.h>
+#include <zephyr/irq.h>
+#include <soc.h>
+
+#define PRCR_KEY   0xA500U
+#define VBATT_PORT 4
+
+static const gpio_pin_t vbatt_pins[] = {
+	2U,
+	3U,
+	4U,
+};
```

Comment:
It seems to be a hardware configuration.
It is good to define in devicetree.

Reply:
It is good to define an array of `vbatt_pins` in the devicetree and reference that.